### PR TITLE
Partial users online (discussion)

### PR DESCRIPTION
**Pull request by aXises**
### Team Monkey pull request checklist (Jungle edition)

-   [ ] Remember to format your code! (`npm run prettier && npm run eslint`)
-   [ ] Remember to run the tests! (`npm run test`)
-   [ ] Don't forget comments and docstrings!
-   [ ] Is it a breaking change? Remember to mention that!

### Description

**PR UP FOR REVIEW PURPOSES**. Comment your thoughts on the implementation and the integration into current systems.

There have been many alternative implementations of this exact same thing made by many developers. A general abstracted implementation was added in #104. 

How should we integrate these systems? Some areas are obsolete however could he salvaged for #96.

🐵🐒🙊🙉🙈

### CODE REVIEW

File: src/client/home/OnlineUsersContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
```
Comment (OwOindie) [inline src/client/home/OnlineUsersContainer.tsx:1]: Consider a redux pattern for this type of systems. Pure functions are important if the data is constantly changing, especially in this environment.

I can help with setting up the action/reducer flow throughout the app if necessary.

**Review by OwOindie — CHANGES_REQUESTED** [review]
Consider a redux pattern for this type of systems. Pure functions are important if the data is constantly changing, especially in this environment.

I can help with setting up the action/reducer flow throughout the app if necessary.

File: src/client/home/OnlineUsersContainer.tsx (hunk 1 above)
Comment (aXises) [reply src/client/home/OnlineUsersContainer.tsx:1]: Hi Mr OwOindie, thank you for your suggestion and offer to assist us, unfortunately, collaboration is only limited to team members as this is a capstone project.

In response to your pattern suggestion, we have decided to not use redux for our project as it introduces an additional layer of complexity which may cause other important features to be delayed.

Thank you once again for your interest,
Management

**Comment by JZ4012** [comment]
There seems to be a lot of checks that are failing, please resolve those first.

**Pull request closed without merging**